plugins/airbrake/README.md

```diff
@@ -98,6 +98,17 @@ The Airbrake plugin provides connectivity between Backstage and Airbrake (https:
    }
    ```
 
+   ```typescript
```

Comment:
We aren't ready to have everyone use the new backend system as the default. You'll want to keep the existing documentation as is. But then add a new section specifically about this. The Azure DevOps Backend is a good example: https://github.com/backstage/backstage/tree/master/plugins/azure-devops-backend#new-backend-system